oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
```

Comment:
```create_with_claims()`` has no meaning for most google credentials, so I would like to avoid factory method ```create_with_claims(google_credentials, claims)```.  However, ServiceAccountCredentials actually use claims when obtaining an oauth2 access token, so I think it would be reasonable to have a  ``create_with_claims(service_account_credentials, claims)```, and have JWTAccessCredentials extend ServiceAccountCredentials.